frontend/src/metabase/lib/formatting/date.tsx

```diff
@@ -118,8 +118,8 @@ export const DATE_RANGE_FORMAT_SPECS: {
   const DDDoP = "DDDo [days of the year]";
   const DoS = "Do [day of the month]";
   const DoP = "Do [days of the month]";
-  const woS = "wo [week of the year]";
-  const woP = "wo [weeks of the year]";
+  const WoS = "Wo [week of the year]";
```

Comment:
AFAIK this is unreachable code. There cannot be week-of-year ranges in the app, and there is no way to call this function to trigger this behavior. Still, I changed it for consistency.